Dockerfile

```diff
@@ -102,7 +105,7 @@ COPY --from=static_builder --chown=www-data:www-data /app/public /var/www/html/L
 COPY default.conf /etc/nginx/nginx.conf
 
 EXPOSE 80
-VOLUME /conf /uploads /sym /logs
+VOLUME /conf /uploads /sym /logs /image-tmp /image-jobs /extract-jobs

```

Comment:
This feels excessive as they should all be temporary, but :man_shrugging:

Reply:
They probably need permissions checks too

Reply:
Good point on permission checks.

yeah I know it feels excessive, but that way you may run your docker base container on a small instance and have some scratch pad for the tmp files. :)

Reply:
fixed [fe34506](https://github.com/LycheeOrg/Lychee-Docker/pull/208/commits/fe3450651bb19948a5a18043515b20eaa5cb8276)

Reply:
> yeah I know it feels excessive, but that way you may run your docker base container on a small instance and have some scratch pad for the tmp files. :)

But do you need three separate ones? e.g. it could be `/lychee-tmp/image-tmp` etc? That would also save remapping if we add more or rename them again.

It's good enough to merge, just feels a bit much.

Reply:
> But do you need three separate ones? e.g. it could be `/lychee-tmp/image-tmp` etc? That would also save remapping if we add more or rename them again.
> 
> It's good enough to merge, just feels a bit much.

Ohhh, good point, I like it!
I will fix that in v6.

Reply:
Could just cheat it here, but if there's a good place in the v6 too, sure :)

Reply:
[a8864d4](https://github.com/LycheeOrg/Lychee-Docker/pull/208/commits/a8864d45692c721a05543794819e8678cf685eb9)